src/__mocks__/axios.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  get: jest.fn().mockResolvedValue({
```

Comment:
Good idea to mock axios. The downside of the current approach is that the get will only have a single value by default for all tests. This could cause some unexpected behavior. A better approach is to mock specific uses of axios like [this](Please always include some context or instructions in the PR. I understand that the create-react-app boilerplate was used but I wasn't sure where and how to run the backend side of your application.

Also, it helps to understand some decisions you made while developing the project so we can discuss them in more detail

)

Reply:
Thank you! I think the link may have the wrong paste as it's a copy of one of the first comments 🙀 But the link sounds really useful! I'll be sure to check it out!